ydb_sqlalchemy/sqlalchemy/__init__.py

```diff
@@ -364,8 +412,18 @@ def do_commit(self, dbapi_connection) -> None:
         # TODO: needs to implement?
         pass
 
+    def do_executemany(self, cursor, statement, parameters, context=None):
```

Comment:
Can you add typehints for all new/modified methods?

Reply:
Yes, missed it